src/helpers/language.ts

```diff
@@ -4,32 +4,41 @@ import spanishStrings from "./strings-es"
 import dutchStrings from "./strings-nl"
 import germanStrings from "./strings-de"
 import frenchStrings from "./strings-fr"
-import portugueseStrings from "./strings-pt"
+import portugueseBrazilStrings from "./strings-pt"
 import italianStrings from "./strings-it"
 import polishStrings from "./strings-pl"
+import portuguesePortugalStrings from "./strings-pt-pt"
 
-export type Language = "" | "en" | "es" | "nl" | "de" | "fr" | "pt" | "it" | "pl"
+export type Language = "" | "en" | "es" | "nl" | "fr" | "de" | "it" | "pt" | "pt-pt" | "pl"
 
 function format(resolvedString: string, args?: { [key: string]: string }) {
 	if (!resolvedString || !args) return resolvedString;
 	return resolvedString.replace(/\{\s*([^}\s]+)\s*}/g, (_, key) => args[key]);
 }
 
-export function language(key: string, specifiedLanguage?: string, args?: { [key: string]: string }): string {
-	const currentLanguage: Language = getLanguageFromIso(specifiedLanguage || MyDataHelps.getCurrentLanguage());
-
-	let resolvedString = null;
-	if (currentLanguage == "en") resolvedString = englishStrings[key];
-	if (currentLanguage == "es") resolvedString = spanishStrings[key];
-	if (currentLanguage == "nl") resolvedString = dutchStrings[key];
-	if (currentLanguage == "de") resolvedString = germanStrings[key];
-	if (currentLanguage == "fr") resolvedString = frenchStrings[key];
-	if (currentLanguage == "pt") resolvedString = portugueseStrings[key];
-	if (currentLanguage == "it") resolvedString = italianStrings[key];
-	if (currentLanguage == "pl") resolvedString = polishStrings[key];
-	if (resolvedString != null) return format(resolvedString, args);
-
-	return format(englishStrings[key], args);
+export function language(key: string, specifiedLocale?: string, args?: { [key: string]: string }): string {
+	// Normalize formats like pt-PT or pt_PT to a consistent pt-pt for lookup.
+	const currentLocale = (specifiedLocale || MyDataHelps.getCurrentLanguage() || "").replace("_", "-").toLowerCase();
```

Comment:
Need to clean this up a little and add some unit tests, but this should generally make pt-PT and any future country-specific strings tables work.